pkg/authorization/audit_logger.go

```diff
@@ -40,6 +41,10 @@ func NewAuditLogger(auditPrefix string, delegate authorizer.Authorizer) authoriz
 
 	return authorizer.AuthorizerFunc(func(ctx context.Context, attr authorizer.Attributes) (authorizer.Decision, string, error) {
 		dec, reason, err := delegate.Authorize(ctx, attr)
+		// skip logging allow decisions to not flood the audit log
```

Comment:
Do you want to skip adding the audit annotations in this case too (from L54-58)?

Reply:
that was @sttts's OOB proposal, but I can just skip the logs.

Reply:
done